### DISCUSSION TITLE
Change phase name when they are initially the same

### DISCUSSION THREAD

**Discussion by lebraunjames** [mtex-toolbox/mtex#1819] [Ask Anything] [answered]
Hi, I have an ebsd file of a Cobalt alloy with fcc and hcp phases that are both unintentionally named as Co and Co phase. Every other property is different (lattice parameters, symmetry, etc.) This makes it impossible to call for a specific phase and i have no idea how to change it through code (I have to do it manually). Is there a way to make matlab call a specific phase by number (or other properties) and change its name?

Thank you, 
Jubert

**Comment by kilir** [comment, marked as answer]
Hi,
I hope this helps:
```
mtexdata olivine
% rename 1st indexed phase 
ebsd.CSList{2}.mineral = 'Some_Silicate'
```
Cheers,
Rüdiger

**Reply by lebraunjames** [reply]
It absolutely worked, thank you for the quick answer!
